warp10/src/main/java/io/warp10/continuum/gts/GTSHelper.java

```diff
@@ -5101,10 +5102,10 @@ public static void reset(GeoTimeSerie gts) {
     GeoTimeSerie mapped = gts.clone();
 
     //
-    // Do nothing if there are no values and gts was not bucketized
+    // Do nothing if gts was not bucketized and either there are no value or the mapper is a filler mapper
```

Comment:
Better thrown an exception if a macrofiller is used and the GTS is not bucketized.